target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -109,6 +108,26 @@
 					bias-disable;
 				};
 			};
+
+			usb0_pwr_en_pin: usb0_pwr_en_pin {
```

Comment:
"pin" in c2600 dts differs from "pinS" in other boards dts.
If that's not a problem, then everything seems fine :)

Thank you very much, @heinzek for cleaning this stuff altogether :)